pyproject.toml

```diff
@@ -45,4 +45,61 @@ tracker = "https://github.com/maroba/findiff/issues"
 include = ["findiff"]
 
 [tool.setuptools.dynamic]
-version = { attr = "findiff.__version__" }
\ No newline at end of file
+version = { attr = "findiff.__version__" }
+
+[tool.coverage]
+run.source_pkgs = ["findiff", "tests"]
```

Comment:
Why include "tests" in source_pkgs? In the workflow, the command that is called for coverage is:

```
pytest --cov=findiff --cov-report=html tests
```

Reply:
I consider it a best practice because unexecuted lines in a test file smell like a bug.
This can detect function name collisions. (`test_calc_accuracy_from_offsets_symbolic`)